### PR TITLE
Backup agent also use mmsApiKey

### PR DESCRIPTION
**Pull request by allaire**
Was getting:

```
Error loading configuration file /etc/mongodb-mms/backup-agent.config:
    `mmsApiKey` has not been specified in configuration file.
```

### CODE REVIEW

File: attributes/mms_agent.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@
 default['mongodb']['mms_agent']['monitoring']['sslRequireValidServerCertificates'] = false
 
 default['mongodb']['mms_agent']['backup']['version'] = '1.4.6.43-1'
-default['mongodb']['mms_agent']['backup']['apiKey'] = node['mongodb']['mms_agent']['api_key']
+default['mongodb']['mms_agent']['backup']['mmsApiKey'] = node['mongodb']['mms_agent']['api_key']
```
Comment (jspiewak) [inline attributes/mms_agent.rb:20]: What version of the backup agent are you using?
Did MongoDB, Inc change what they use with the recent release of the backup agent?

Comment (allaire) [inline attributes/mms_agent.rb:20]: version 2.1.0

Comment (jamesonjlee) [inline attributes/mms_agent.rb:20]: http://mms.mongodb.com/help/reference/backup-agent/#mmsApiKey it appears as so

**Comment by jamesonjlee** [comment]
@allaire could you also bump the version to latest so this config works with the agent?

**Comment by allaire** [comment]
Done

**Pull request merged**